src/main/kotlin/uk/gov/justice/digital/hmpps/createandvaryalicenceapi/service/LicenceService.kt

```diff
@@ -220,6 +220,20 @@ class LicenceService(
       notifyReApprovalNeeded(licenceEntity)
     }
 
+    if (request.status === APPROVED && licenceEntity is HardStopLicence) {
+      val com = licenceEntity.responsibleCom
+      if (com != null) {
+        notifyService.sendHardStopLicenceApprovedEmail(
+          com.email,
+          licenceEntity.forename ?: "unknown",
+          licenceEntity.surname ?: "unknown",
+          licenceEntity.crn,
+          licenceEntity.conditionalReleaseDate ?: licenceEntity.actualReleaseDate,
```

Comment:
Unsure about what the release date should be in this instance? Is it still CRD if set otherwise ARD?

Reply:
think this is right, could use licenceStartDate??? @SebNegahbanMoJ what do you think?

Reply:
If we were going to be explicit it should be ARD ?: CRD, as CRD should always be set and ARD is the `confirmed` date.
Given that's the case though, yeah we can probably use `licenceStartDate`